includes/Admin/Admin_Page.php

```diff
@@ -63,6 +64,29 @@ public function render_page() {
 
 		$available_plugins = $this->get_available_plugins();
 
+		$selected_plugin_basename = filter_input( INPUT_GET, 'plugin', FILTER_SANITIZE_STRING );
+
 		require_once WP_PLUGIN_CHECK_PLUGIN_DIR_PATH . '/templates/admin-page.php';
 	}
+
+	/**
+	 * Add "check this plugin" link in the plugins list table.
+	 *
+	 * @param array  $actions     List of actions.
+	 * @param string $plugin_file Plugin main file.
+	 * @return array
+	 */
+	public function filter_plugin_action_links( $actions, $plugin_file ) {
+
+		if ( is_plugin_active( $plugin_file ) ) {
+
+			$actions[] = sprintf(
+				'<a href="%1$s">%2$s</a>',
+				esc_url( admin_url() . 'tools.php?page=plugin-check&plugin=' . $plugin_file ),
+				esc_html__( 'Check this plugin', 'plugin-check' )
+			);
```

Comment:
Maybe just to be safe, we should make sure to only add this link if the user has access to the plugin check page (i.e. has the `activate_plugins` capability, see https://github.com/10up/plugin-check/pull/77/files#diff-8f134a3d2e0f406fc9691e5163d91e0fabef5e4446a7f4c20da8cb412ef56b3eR31).

Reply:
Done